docs/reference/commandline/kill.md

```diff
@@ -19,3 +19,9 @@ parent = "smn_cli"
 
 The main process inside the container will be sent `SIGKILL`, or any
 signal specified with option `--signal`.
+
+> **Note:**
+> `ENTRYPOINT` and `CMD` in the *shell* form will be started as a
+> subcommand of `/bin/sh -c`, which does not pass signals. This means
+> that the executable will not be the container’s PID 1 - and will
+> not receive Unix signals.
```

Comment:
Thanks for the comment @thaJeztah, I updated the file to reflect your suggestions.